site/src/atomics/aspect-ratio.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Aspect ratio
+description: Aspect ratio related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - aspect-ratio
+---
+
+# Aspect Ratio Atomics
+
+Aspect ratio atomics are used to set the width-to-height ratio of an element, even if the parent container or viewport size changes.
+
+| cssproperty  | ratio                               | screensize |
+| ------------ | ----------------------------------- | ---------- |
+| aspect-ratio | `1-1`, `2-1`, `4-3`, `9-16`, `16-9` | N/A        |
+
+## Usage
+
+Here are examples of aspect ratio atomics being applied to `video` and `img` elements. When used on `video` or `iframe` elements with the matching aspect ratio, it will remove the black bars and scale responsively.
```

Comment:
Can we add a little more context on what "the black bars" are and why they occur in the first place?